make-all.sh

```diff
@@ -27,3 +27,12 @@ fi
 cp src/TerminalFighter/TerminalFighter ../
 ./test/runTests
 cd ..
+
+# Update docker if there are any new files
+docker build . -t terminal_documentation
+
+# "doxygen -g" will create a default config file named Doxyfile
+# "patch < Doxyfile.patch will apply the configuration patch
+#sudo docker run --rm -v `pwd`:/documentation test1 /bin/bash -c "doxygen -g && \
+    #patch < Doxyfile.patch && doxygen Doxyfile
+docker run --rm -v `pwd`:/documentation terminal_documentation /bin/bash -c "doxygen Doxyfile"
```

Comment:
`doxygen Doxyfile` isn't complicated so we don't need to invoke it as a bash specific command.
```
docker run --rm -v `pwd`:/documentation terminal_documentation doxygen Doxyfile
```
should be fine